src/main/java/nl/novi/spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,42 @@
+package nl.novi.spring_boot_tech_it_easy_controller.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class TelevisionsController {
```

Comment:
Al je paden beginnen met "/televisions", daarom is het misschien handig om hier, boven je klasse, de `@RequestMapping("/televisions")` te gebruiken. Zo hoef je dat niet bij elke mapping opnieuw te definieren. Als je straks besluit om toch bananen in plaats van televisies te verkopen, dan hoef je dat nu maar op 1 plek aan te passen.